src/streaming/MediaPlayer.js

```diff
@@ -1920,6 +1920,33 @@ function MediaPlayer() {
     }
 
     /**
+     * Get the value of useDeadTimeLatency in AbrController. @see setUseDeadTimeLatencyForAbr
+     *
+     * @returns {boolean=}
+     *
+     * @memberof module:MediaPlayer
+     * @instance
+     */
+    function getUseDeadTimeLatencyForAbr() {
+        return abrController.getUseDeadTimeLatency();
+    }
+
+    /**
+     * Set the value of useDeadTimeLatency in AbrController. If true, only the download
+     * portion will be considered part of the download bitrate and latency will be
+     * regarded as static. If false, the reciprocal of the whole transfer time will be used.
+     * Defaults to true.
+     *
+     * @param {boolean=} useDeadTimeLatency - True or false flag.
+     *
+     * @memberof module:MediaPlayer
+     * @instance
+     */
+    function setUseDeadTimeLatencyForAbr(useDeadTimeLatency) {
+        abrController.setUseDeadTimeLatency(useDeadTimeLatency);
+    }
```

Comment:
There's a lot of config that is stored in MediaPlayerModel and not reset during the player lifetime (see https://github.com/Dash-Industry-Forum/dash.js/blob/development/src/streaming/models/MediaPlayerModel.js#L432-L433). 
It might make sense to move everything there (and maybe rename it to MediaPlayerConfig). 
I find it generally a good idea to have such a store used only to store configuration, and usable anytime during the player's lifetime (other issue with abrController config params is that trying to set them before calling `initialize` throws an exception, which does not make a lot of sense when setting configuration)